subsys/bluetooth/controller/ll_sw/ull_sched.c

```diff
@@ -571,15 +644,19 @@ static uint8_t after_match_slot_get(uint8_t user_id, uint32_t ticks_slot_abs,
 {
 	uint32_t ticks_to_expire_prev;
 	uint32_t ticks_slot_abs_prev;
+	uint32_t ticks_anchor_prev;
 	uint32_t ticks_to_expire;
 	uint8_t ticker_id_prev;
 	uint8_t ticker_id;
+	uint8_t retry;
 
 	ticks_slot_abs += HAL_TICKER_US_TO_TICKS(EVENT_JITTER_US << 3);
 
 	ticker_id = ticker_id_prev = TICKER_NULL;
+	ticks_anchor_prev = 0U;
 	ticks_to_expire = ticks_to_expire_prev = 0U;
 	ticks_slot_abs_prev = 0U;
+	retry = 4U;
```

Comment:
I think the code would benefit from having a comment w.r.t. why `4` was chosen for `retry`.

Reply:
done